### DISCUSSION TITLE
srt-live-transmit

### DISCUSSION THREAD

**Discussion by stevespaw** [OvenMediaLabs/OvenMediaEngine#1501] [Help]
I have been using OME for some time with SRT from OBS and others - no issues. Now I have a need for basic srt-live-transmit from udp multicast. It seems to connect but never shows video and crashes my other streams until I disconnect it. I may have something to do with how I format the command. 
is there an example? I am new to srt-live-transmit.
Here is what I am trying:
srt-live-transmit udp://172.30.0.155@224.1.1.1:11005 srt://ingest1.tv:9999?mode=caller&latency=200&streamid=srt%3A%2F%2Fingest1.tv%3A9999%2Fapp%2Fout4

Any ideas?

**Comment by basisbit** [comment]
Try latency=200000 (for 200 ms)

**Comment by stevespaw** [comment]
Thanks, but that did not help. Any examples of srt-live-transmit with the proper stream ID for OME?
